### PR TITLE
[RFR] [MIG] 8.0 account analytic distribution

### PR DESCRIPTION
**Pull request by PieterPaulussen**
Currently working on the tests

### CODE REVIEW

**Comment by PieterPaulussen** [comment]
@StefanRijnhart @lembregtse I restored functionality for the module to be compatible with 8.0. I also updated the tests.

File: account_analytic_distribution/README.rst [hunk 1]
```diff
@@ -48,14 +48,10 @@ in invoice lines.
...
 
-You can continue using 'Analityc Account' field, but 'Analytic Distribution'
+You can continue using 'Analytic Account' field, but 'Analytic Distribution'
 has precedence if defined.
 
 
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
-   :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/87/9.0
-
```
Comment (StefanRijnhart) [inline account_analytic_distribution/README.rst]: Keep this but replace the version number in the target

File: account_analytic_distribution/__openerp__.py [hunk 2]
```diff
@@ -7,23 +7,24 @@
...
+        "views/account_config_settings.xml",
+        "views/account_invoice.xml",
+        "views/account_move.xml",
+        "views/account_move_line.xml",
     ],
     "installable": True,
     "application": False,
 }
+
+# not compatible with account analytic plan
```
Comment (StefanRijnhart) [inline account_analytic_distribution/__openerp__.py]: I know I suggested this, but according to the README it is!

File: account_analytic_distribution/__init__.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
-# -*- coding: utf-8 -*-
-# Copyright 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-
```
Comment (StefanRijnhart) [inline account_analytic_distribution/__init__.py]: PLease don't remove credits. I would not even claim copyright on migrations of a mechanical nature. Adding your name to the contributor's list is enough.

Comment (PieterPaulussen) [reply account_analytic_distribution/__init__.py]: Wasn't my intention at all. Sorry. It's the default file header for newly created python files on my system. I was just focusing on functionality. ;-)

Comment (lembregtse) [reply account_analytic_distribution/__init__.py]: The sorry broke my heart.

**Comment by StefanRijnhart** [comment]
Don't you want to propose into OCA?

**Comment by StefanRijnhart** [comment]
You have made review a little bit more complicated by renaming views and model files.

File: account_analytic_distribution/views/account_move_line_view.xml [hunk 4]
```diff
@@ -2,40 +2,42 @@
...
-    </record>
+        </record>
 
-    <record model="ir.ui.view" id="view_move_line_tree">
-        <field name="name">Add Analytic distribution column</field>
-        <field name="model">account.move.line</field>
-        <field name="inherit_id" ref="account.view_move_line_tree"/>
-        <field name="arch" type="xml">
-            <field name="analytic_account_id" position="before">
-                <field name="analytic_distribution_id" groups="analytic.group_analytic_accounting" einvisible="not context.get('analytic_journal_id', False)"/>
```
Comment (PieterPaulussen) [inline account_analytic_distribution/views/account_move_line_view.xml]: is 'einvisible' a possible typo? I can't seem to find any reference to it.

Comment (StefanRijnhart) [reply account_analytic_distribution/views/account_move_line_view.xml]: Must be the case, indeed.

**Review by lembregtse — APPROVED** [review]

**Pull request merged**